### PR TITLE
Restore style guide docs in taxonomy

### PR DESCRIPTION
**Pull request by bradleycamacho**
### Tell us why
There were some missing docs in the style guide taxonomy. This PR restores them.

### CODE REVIEW

File: src/nav/style-guide.yml [hunk 1]
```diff
@@ -58,6 +58,8 @@ pages:
         path: /docs/style-guide/writing-guidelines/formatting-terminal-commands
   - title: Processes and procedures
     pages:
+      - title: Tech writer workflow
```
Comment (austin-schaefer) [inline src/nav/style-guide.yml:61]: This is called `Tech writer workflow` here, but `Tech Writer Workflow` in the actual page frontmatter.

File: src/nav/style-guide.yml [hunk 2]
```diff
@@ -74,11 +76,19 @@ pages:
         path: /docs/style-guide/processes-procedures/publication-checklist
       - title: Embed videos
         path: /docs/style-guide/processes-procedures/embed-videos
+      - title: Update left navigation pane
```
Comment (austin-schaefer) [inline src/nav/style-guide.yml:79]: Similar teeny nitpick: This is called `left navigation pane` in sidebar but `left-navigation pane` in doc title. The no hyphen version is more correct IMO

**Review by austin-schaefer — CHANGES_REQUESTED** [review]
Approved, a few minor tweaks/suggestiosn

**Comment by bradleycamacho** [comment]
Closed since this work has been done in other PRs.

**Pull request closed without merging**